server/src/rsvp/rsvp.service.ts

```diff
@@ -53,13 +54,35 @@ export class RsvpService {
 		return null;
 	}
 
+	/**
+	 * Attempt to create a new RSVP data entry, if it already exists just update it
+	 * @param guest The primaryData of the guest to add RSVP info for
+	 * @param rsvp The new or updated RSVP info
+	 * @return The resulting RSVP data
+	 */
+	async createOrUpdate(
```

Comment:
did you implement this twice?

Reply:
nvm I think it’s slightly different. Same question as above, then

Reply:
No but I created the new function in both rsvp.service and contact.service